students/brian_warn/session02/series.py

```diff
@@ -0,0 +1,54 @@
+#!/Library/Frameworks/Python.framework/Versions/3.6/bin/python
+# series.py
+
+def main():
+    def fibonacci(n):
+        ''' Compute fibonacci series for a given input number
+        :arg input integer
+        :return: nth value in the fibonacci series
+        '''
+        if n == 0:
+            return 0
+        if n == 1:
+            return 1
+        return fibonacci(n - 1) + fibonacci(n - 2)
+
+    def lucas(n):
+        ''' Compute Lucas Numbers series for a given input number
+        :arg input integer
+        :return: nth value in the Lucas Numbers series
+        '''
+        if n == 0:
+            return 2
+        if n == 1:
+            return 1
+        return luc(n - 1) + luc(n - 2)
+
+    def sum_series(n, a = 0, b = 1):
+        ''' Compute Lucas Numbers series or Fibonacci series based upon optional parameter values
+        :arg 
+            n = input integer
+            a = first optional argument
+            b = second optional argument
+        :return: nth value in either the Lucas Numbers series or Fibonacci series
+        '''
+        if a == 0 and b == 1:
+            result = 'Fibonacci Sequence Result for ' + str(n) + ' is :' + fibonacci(n)
```

Comment:
without running the code, I think I see the error.

you are trying to add together a bunch of strings, which wokds fine. But that last one is a problem:

```
  + fibonacci(n)
```
that function call returns an integer, not a string, so you can't add it to a string.

try wrapping it in a str() call, like you did for the "n" earlier.

Or better yet, replace that line with string formatting :-)